elastic/apm-server
# record per-agent config status

## Motivation/summary

~~Record the clientIP, service name, service
environment, and etag for the applied config, for
each agent request. This can be used to see the
current state of config application, convergence
towards a new config, or if there are services
that are running old config versions.~~

Record the etag for applied agent_configs in elastic search.

closes https://github.com/elastic/apm-server/issues/5470

## Checklist




For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

Running apm-server under fleet, start an agent that requests its config from apm-server. Update the config, and verify that the timeseries has been ingested into elasticsearch.
